packages/webapp/src/components/Pages/DataShowcasePage/hooks/useOnGetAllAssetsDownloadInfo.ts

```diff
@@ -0,0 +1,280 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import {
+	FileCollection,
+	FileWithPath,
+} from '@data-wrangling-components/utilities'
+import { useCallback } from 'react'
+import type { IEvaluateResult } from 'sds-wasm'
+
+import {
+	getMetricsByCountLabels,
+	getMetricsByLenLabels,
+} from '~components/Charts/hooks'
+import type { DownloadInfo } from '~components/controls/DownloadButton'
+import {
+	getAggregatesCsv,
+	getAnalysisByCountCsv,
+	getAnalysisByLenCsv,
+	getMetricsSummaryCsv,
+} from '~components/DataEvaluationInfoDownloader'
+import { getMicrodataMetricsItems } from '~components/MetricsSummaryTable'
+import type { IContextParameters } from '~models';
+import { AggregateType } from '~models'
+import {
+	useAllContextsParametersValue,
+	useSensitiveContentValue,
+	useWasmWorkerValue,
+} from '~states'
+import type { SdsWasmWorker } from '~workers/sds-wasm'
+
+import { useGetSyntheticCsvContent } from '../DataSynthesis/hooks'
+
+async function generateAggregatesCsv(
+	worker: SdsWasmWorker,
+	c: IContextParameters,
+): Promise<FileWithPath[]> {
+	return [
+		new FileWithPath(
+			new Blob(
+				[await getAggregatesCsv(worker, c.key, AggregateType.Sensitive)],
+				{
+					type: 'text/csv',
+				},
+			),
+			`${c.key}/${AggregateType.Sensitive}_aggregates.csv.csv`,
+			`${c.key}/${AggregateType.Sensitive}_aggregates.csv.csv`,
```

Comment:
There are several places where the file extension is .csv.csv. I am guessing it should be one .csv but let me know if I am missing something,